CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# How to contribute
+
+Operator Controller is an Apache 2.0 licensed project and accepts contributions via GitHub pull requests (PRs).
+
+This document outlines some conventions on commit message formatting, contact points for developers, and other resources
+to help Operator Controller contributors.
+
+## Communication
+
+- Email: [operator-framework-olm-dev](mailto:operator-framework-olm-dev@googlegroups.com)
+- Slack: [#olm-dev](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [olm-gg](https://groups.google.com/g/operator-framework-olm-dev)
+- Weekly in Person Working Group Meeting: [olm-wg](https://github.com/operator-framework/community#operator-lifecycle-manager-working-group) 
+
+## Getting started
+
+- Fork the repository on GitHub.
+- Clone onto your local development machine via `git clone https://github.com/{$GH-USERNAME}/operator-controller`.
+- Add the operator-framework/operator-controller as an upstream.
+  remote: `git remote add upstream https://github.com/operator-framework/operator-controller`.
+- Create a new branch from the default `main` branch and begin development.
+
+## Contribution flow
+
+Any new contribution should be accompanied by a new or existing issue. This issue can help track work, discuss the
+design and implementation, and help avoid wasted efforts of multiple people working on the same issue. Trivial changes,
+like fixing a typo in the documentation, do not require the creation of a new issue.
+
```

Comment:
Should we have some guidelines on how to create and tag issues for features, bugs or security vulnerabilities?

Reply:
Might be a better use of our time to create a few [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository).

Reply:
Captured this idea in an issue: https://github.com/operator-framework/operator-controller/issues/154